includes/settings/js/src/components/fields/EmailSettings.jsx

```diff
@@ -0,0 +1,90 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/react";
+import React from "react";
+import Icon from "../../../../../components/icons";
+import { __ } from "@wordpress/i18n";
+
+// TODO-abotz: figure out why we need important here and remove
+const emailDomainTextArea = css({
+	maxWidth: "none !important",
+	minHeight: "55px !important",
+});
```

Comment:
Inline Emotion styles will be overridden by more specific external CSS by default (in this case the `.atlas-content-modeler-admin-page textarea` styles in our `_forms.scss` are overriding the Emotion styles).

To work around this you could use the `css` JS template string and write your CSS as normal but with a `.atlas-content-modeler-admin-page` prefix:

```suggestion
const emailDomainTextArea = css`
	.atlas-content-modeler-admin-page & {
		max-width: none;
		min-height: 55px;
	}
`;
```

The `&` there will be replaced by the class that Emotion generates automatically, resulting in a CSS rule like this that overrides the `.atlas-content-modeler-admin-page textarea` styling:

```css
.atlas-content-modeler-admin-page .css-bkv9i3-EmailSettings {
    max-width: none;
    min-height: 55px;
}
```

It's not great that we have to do this in some cases, but it could be reduced over time if we cut down on generic/global Sass and CSS.

Reply:
Thanks for the explanation man, makes sense!